composer.json

```diff
@@ -9,8 +9,8 @@
         }
     ],
     "require": {
-        "illuminate/support": "^4.2|^5|^6|^7|^8|^9",
-        "illuminate/cache": "^4.2|^5|^6|^7|^8|^9"
+        "illuminate/support": "*",
```

Comment:
we have a constraint of `>4.2` and cannot guarantee this package works after v9.